src/components/WebexLocalMedia/WebexLocalMedia.js

```diff
@@ -16,30 +16,37 @@ import './WebexLocalMedia.scss';
  * @param {string} props.meetingID  ID of the meeting from which to obtain local media
  * @returns {Object} JSX of the component
  */
-export default function WebexLocalMedia({meetingID}) {
+export default function WebexLocalMedia({className, meetingID}) {
   const [mediaRef, {width}] = useElementDimensions();
   const {localVideo} = useMeeting(meetingID);
   const {ID} = useMe();
   const videoRef = useStream(localVideo);
 
-  const classBaseName = 'local-media';
-  const classObjects = {};
+  const classBaseName = `${WEBEX_COMPONENTS_CLASS_PREFIX}-local-media`;
+  const mainClasses = {
+    [classBaseName]: true,
+    desktop: width >= PHONE_LARGE,
```

Comment:
Why is this not a full class name like in In-Meeting? e.g. [`${classBaseName}-desktop`]

Reply:
I was just using the previous names, I will update